java/server/src/main/java/com/squareup/subzero/server/ServerExceptionMapper.java

```diff
@@ -0,0 +1,14 @@
+package com.squareup.subzero.server;
+
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+
+public class ServerExceptionMapper implements ExceptionMapper<Throwable> {
+  public Response toResponse(Throwable exception) {
+    return Response.status(400)
```

Comment:
500?

Reply:
It could be either (e.g. bad parameters). We don't really care since this is all a debug server and we just want to show some useful error message. The full stack trace is displayed in the logs.